github/github.go

```diff
@@ -648,7 +648,7 @@ type TwoFactorAuthError ErrorResponse
 func (r *TwoFactorAuthError) Error() string { return (*ErrorResponse)(r).Error() }
 
 // RateLimitError occurs when GitHub returns 403 Forbidden response with a rate limit
-// remaining value of 0, and error message starts with "API rate limit exceeded for ".
+// remaining value of 0
```

Comment:
nit: please add the ending period ('.') back in for the auto-generated godocs.

Reply:
Added the period.